core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -524,6 +529,15 @@ class SparkContext(config: SparkConf) extends Logging {
     executorEnvs ++= _conf.getExecutorEnv
     executorEnvs("SPARK_USER") = sparkUser
 
+    val redactionRegex = _conf.get(SECRET_REDACTION_PATTERN)
+    _shuffleDriverComponents = ShuffleDataIOUtils.loadShuffleDataIO(config).driver()
+    _shuffleDriverComponents.initializeApplication().asScala.foreach { case (k, v) =>
+      val key = ShuffleDataIOUtils.SHUFFLE_SPARK_CONF_PREFIX + k
+      _conf.set(key, v)
+        logDebug(s"Setting shuffle spark configuration ${key} to " +
```

Comment:
Indentation is off. But really I'm still not seeing the benefit of this log message.

Reply:
yeah, I agree.  Sorry my fault for suggesting this earlier, I misread the flow of getting this into the EnvironmentUpdate event.  Individual plugins can log their confs if they want.